src/include/duckdb/main/connection.hpp

```diff
@@ -115,6 +115,11 @@ class Connection {
 		UDFWrapper::RegisterFunction(name, args, ret_type, function, *context);
 	}
 
+	template<typename TR, typename... Args>
+	void CreateVectorizedFunction(string name, scalar_function_t udf_func) {
```

Comment:
Could you add support here for vararg functions as well and add a test case for them?